fly.toml

```diff
@@ -0,0 +1,43 @@
+# fly.toml app configuration file generated for nijikai-go on 2025-03-19T13:34:15+09:00
+#
+# See https://fly.io/docs/reference/configuration/ for information about how to use this file.
+#
+
+app = 'nijikai-go'
+primary_region = 'nrt'
+console_command = '/rails/bin/rails console'
+
+[build]
+
+[deploy]
+  release_command = "./bin/rails db:prepare"
+
+[http_service]
+  internal_port = 3000
+  force_https = true
+  auto_stop_machines = 'stop'
+  auto_start_machines = true
+  min_machines_running = 1
+  processes = ['app']
+
+  [[http_service.checks]]
+    interval = '10s'
+    timeout = '2s'
+    grace_period = '5s'
+    method = 'GET'
+    path = '/up'
+    protocol = 'http'
+    tls_skip_verify = false
+
+    [http_service.checks.headers]
+      X-Forwarded-Proto = 'https'
```

Comment:
Health Checks
- [Health Checks · The Ruby Dispatch](https://fly.io/ruby-dispatch/health-checks/)
- [App configuration (fly.toml) - http_service.checks](https://fly.io/docs/reference/configuration/#http_service-checks)